tests/istioUtils.sh

```diff
@@ -25,9 +25,14 @@ function create_rule() {
       || error_exit 'Could not create rule'
 }
 
+function replace_rule() {
+    ${ISTIOCLI} -n ${NAMESPACE} replace -f ${1} \
+      || error_exit 'Could not create rule'
+}
+
 function cleanup_all_rules() {
     print_block_echo "Cleaning up rules"
-    local rules=($(${ISTIOCLI} -n ${NAMESPACE} list route-rule \
+    local rules=($(${ISTIOCLI} -n ${NAMESPACE} get route-rule \
```

Comment:
When we test PR from other module: 
- Jenkins update istio.VERSION.qual to point to the built artifacts from the module PR and create commit for every change
- Jenkins calls scripts/update_version.sh -Q (reads istio.VERSION.qual) and create a commit with the changes. 

When we start the test everything is set already.